app/javascript/dashboard/components/ChatList.vue

```diff
@@ -449,6 +470,9 @@ export default {
         this.resetAndFetchData();
       }
     },
+    chatLists() {
+      this.chatList = this.conversationList;
```

Comment:
@iamsivin, any reason we could avoid this? Can we use `conversationList` instead of `chatList`

Reply:
@nithindavid If we use the conversationList watcher faced a warning "You may have an infinite update loop in watcher with the expression conversationList", that's the reason used chatLists getter as the watch method.

Reply:
No for line 372, can we use `conversationList`